dev-lisp/clisp/clisp-2.49.95~git.recipe

```diff
@@ -0,0 +1,173 @@
+	1999-2000 Bruno Haible, Sam Steingold
+	2001-2018 Sam Steingold, Bruno Haible
+	"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="6024adde90ea7b8cffb70fd209c282ead6aa19c5"
+SOURCE_URI="https://gitlab.com/gnu-clisp/clisp/-/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="fc865c36416852d27d69b09bc32cc9d1895347e0b1f7d9b7ec4d37de8cb0394f"
+SOURCE_DIR="clisp-$srcGitRev"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+requires_base="
+	clisp$secondaryArchSuffix == $portVersion base
+	haiku$secondaryArchSuffix
+	"
+PROVIDES="
+	clisp$secondaryArchSuffix = $portVersion
+	cmd:clisp = $portVersion
+	cmd:clisp_link = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	libffcall$secondaryArchSuffix
+	libiconv$secondaryArchSuffix
+	libintl$secondaryArchSuffix
+	libncurses$secondaryArchSuffix
+	libreadline$secondaryArchSuffix >= 8
+	libsigsegv$secondaryArchSuffix
+	libunistring$secondaryArchSuffix
+	"
+PROVIDES_berkeley_db="
+	clisp_berkeley_db$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_berkeley_db="
+	$requires_base
+	lib:libdb$secondaryArchSuffix
+	"
+PROVIDES_gdbm="
+	clisp_gdbm$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_gdbm="
+	$requires_base
+	lib:libgdbm$secondaryArchSuffix
+	"
+PROVIDES_libsvm="
+	clisp_libsvm$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_libsvm="
+	$requires_base
+	lib:libsvm$secondaryArchSuffix
+	"
+#PROVIDES_pari="
+#	clisp_pari$secondaryArchSuffix = $portVersion
+#	"
+#REQUIRES_pari="
+#	$requires_base
+#	lib:libpari$secondaryArchSuffix
+#	pari_data$secondaryArchSuffix
+#	"
+PROVIDES_pcre="
+	clisp_pcre$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_pcre="
+	$requires_base
+	lib:libpcre$secondaryArchSuffix
+	"
+PROVIDES_postgresql="
+	clisp_postgresql$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_postgresql="
+	$requires_base
+	lib:libldap$secondaryArchSuffix
+	lib:libpq$secondaryArchSuffix
+	"
+PROVIDES_zlib="
+	clisp_zlib$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_zlib="
+	$requires_base
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libdb$secondaryArchSuffix
+	devel:libffcall$secondaryArchSuffix
+	devel:libgdbm$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libldap$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libpcre$secondaryArchSuffix
+	devel:libpq$secondaryArchSuffix < 12.0
+	devel:libreadline$secondaryArchSuffix >= 8
+	devel:libsigsegv$secondaryArchSuffix
+	devel:libsvm$secondaryArchSuffix
+	devel:libunistring$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+#	devel:libpari$secondaryArchSuffix
+#	pari_data$secondaryArchSuffix
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:cmp
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:msgfmt
+	cmd:sed
+	"
+
+last_prefix=$prefix
+
+BUILD()
+{
+	./configure \
+		FORCE_UNSAFE_CONFIGURE=1 \
+		CXXFLAGS="_D_BSD_SOURCE" \
+		LDFLAGS="-lbsd -lnetwork" \
+		--with-included-regex \
+		--prefix=$last_prefix \
```

Comment:
How is this different then using --prefix=$prefix (you already set that with last_prefix)?

Reply:
I keep the lastly set prefix in `$last_prefix` because in `BUILD()` I use / set this variable to patch `Makefile` to produce different `prefix` for different modules.